src/types.ts

```diff
@@ -23,4 +23,23 @@ export interface InputWithLabelProps {
 export interface TodoListItemProps {
     item: Todo;
     removeTodo: (id: number) => void;
+}
+
+export type SortToggleProps = {
+    isAscending: boolean;
+    onToggle: () => void;
+};
+
+export interface SortingProps {
+    sortMode: 'alphabetic' | 'time';
```

Comment:
super nit: this could be defined as its own type. that way the enum is referenced in the other component and throughout the sorting callbacks. very minor detail in this case, but could be good to keep in mind when complexity increases.
```ts
enum SortModes {
  ALPHABETIC = 'alphabetic'
  TIME = 'time'
}
```